packages/language-server/src/xml-view-diagnostics.ts

```diff
@@ -56,9 +55,13 @@ function validationIssuesToLspDiagnostics(
       case "UnknownTagName":
       case "InvalidAggregationCardinality":
       case "InvalidAggregationType":
+        return {
+          ...commonDiagnosticPros,
+        };
       case "NonStableIDIssue":
         return {
           ...commonDiagnosticPros,
+          code: 666,
```

Comment:
- extract to const,
- expose from validations package?
- maybe expose from messages package we will create in the future?

Reply:
I'll add & expose error code for all validations in a different PR